pkg/aws/policies.go

```diff
@@ -110,6 +110,16 @@ func (c *awsClient) EnsureRole(name string, policy string, permissionsBoundary s
 		}
 	}
 
+	// verify that the paths are the same
+	existingRolePath := ""
+	if output.Role.Path != nil {
+		existingRolePath = *output.Role.Path
+	}
+	if existingRolePath != path {
+		return "", fmt.Errorf("Role with same name but different path exists. Existing role ARN: %s",
+			*output.Role.Arn)
+	}
+
```

Comment:
I think we can shorten this to something like 
```
	if aws.StringValue(output.Role.Path) != path {
		return "", fmt.Errorf("role with same name but different path exists. Existing role ARN: %s",
			*output.Role.Arn)
	}
```

Reply:
fixed